halibot/halibot.py

```diff
@@ -89,13 +93,13 @@ def add_instance(self, name, inst):
 		inst.init()
 		self.log.info("Instantiated object '" + name + "'")
 
-	def _load_config(self):
-		with open(os.path.join(self.workdir, "config.json"), "r") as f:
+	def _load_config(self, configfile="config.json"):
+		with open(os.path.join(self.workdir, configfile), "r") as f:
 			self.config = json.loads(f.read())
 			halibot.packages.__path__ = self.config.get("package-path", [])
 
-	def _write_config(self):
-		with open(os.path.join(self.workdir, "config.json"), "w") as f:
+	def _write_config(self, configfile="config.json"):
```

Comment:
Should we pull out the `"config.json"` string and make it a constant?

Reply:
Late to the party here, but I'll update this PR when #126, #113, etc are merged. This line probably won't apply with those changes.